branding/css/theme.less

```diff
@@ -475,6 +475,7 @@ h1, h2, h3, h4, h5, h6 {
 .alert {
   margin: 18px 0;
   padding-right: 34px;
+  white-space: pre-line;
```

Comment:
@ncounter Do you think this will pose any problems in any other views?

Reply:
@mackdk While I am fine with the rule itself (the definition is `pre-line: Sequences of whitespace will collapse into a single whitespace. Text will wrap when necessary, and on line breaks`) I actually wonder what is the original reason why you added the rule.
What's the case you figured out you need it and how it would look like without it? Maybe we always missed that and it will look fine everywhere, but `alert` messages are dynamically populated and I cannot tell in advance which one will be hit by this. On top of this, IIRC we have alert messages with new lines already, it would be nice to figure out how they are handled and then align one to the other behavior.
I'd be fine adopting the rule, could you please check first how it is done in other places and see if it doesn't break anything. Thank you

Reply:
Thanks to your comments I had a second look, and even if I could not find any explicit multi-line alert, I figured out that we are using JSX for formatting the message (which, as a rusty backend developer I was not aware of). So I reverted the change in the CSS and used JSX as well to show the list of errors. Now the message looks like this:

![Screenshot from 2022-01-28 11-20-16](https://user-images.githubusercontent.com/2292684/151536327-cef0fa7e-33a2-40ff-9009-2ab7f0d322a6.png)

Let me know if the code can be further improved.

Reply:
Nice, well done. I think it looks good now :+1: